staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -128,8 +139,12 @@ func NewCmdAutoscale(f cmdutil.Factory, ioStreams genericiooptions.IOStreams) *c
 	cmd.Flags().Int32Var(&o.Min, "min", -1, "The lower limit for the number of pods that can be set by the autoscaler. If it's not specified or negative, the server will apply a default value.")
 	cmd.Flags().Int32Var(&o.Max, "max", -1, "The upper limit for the number of pods that can be set by the autoscaler. Required.")
 	cmd.MarkFlagRequired("max")
-	cmd.Flags().Int32Var(&o.CPUPercent, "cpu-percent", -1, "The target average CPU utilization (represented as a percent of requested CPU) over all the pods. If it's not specified or negative, a default autoscaling policy will be used.")
+	cmd.Flags().Int32Var(&o.CPUPercent, "cpu-percent", -1, "Flag deprecated: use --cpu instead with percentage or milliCPU value. The target average CPU utilization (represented as a percent of requested CPU) over all the pods. If it's not specified or negative, a default autoscaling policy will be used.")
```

Comment:
Don't change this line, the `MarkDeprecated` is sufficient because it will hide this flag from `--help` output and produce the necessary information for users still relying on that flag.